va/Android.mk

```diff
@@ -44,57 +39,38 @@ LOCAL_SRC_FILES := \
 	va_fool.c
 
 LOCAL_CFLAGS_32 += \
-	-DANDROID \
 	-DVA_DRIVERS_PATH="\"$(LIBVA_DRIVERS_PATH_32)\"" \
-	-DLOG_TAG=\"libva\"
 
 LOCAL_CFLAGS_64 += \
-	-DANDROID \
 	-DVA_DRIVERS_PATH="\"$(LIBVA_DRIVERS_PATH_64)\"" \
-	-DLOG_TAG=\"libva\"
 
-# Android Jelly Bean defined ALOGx, older versions use LOGx
-ifeq ($(ALOG_VERSION), $(ALOG_VERSION_REQ))
-LOCAL_CFLAGS += -DANDROID_ALOG
-else
-LOCAL_CFLAGS += -DANDROID_LOG
-endif
+LOCAL_CFLAGS := \
+	$(if $(filter user,$(TARGET_BUILD_VARIANT)),,-DENABLE_VA_MESSAGING) \
+	-DLOG_TAG=\"libva\"
```

Comment:
Why not? It's just for debug build.

Besides, what trace should I see? Except these I didn't see other log from libva:

04-23 22:04:40.498  1453  3028 I libva   : VA-API version 0.40.0
04-23 22:04:40.498  1453  3028 I libva   : Open DRM devices /dev/dri/card0 for (null)
04-23 22:04:40.498  1453  3028 I libva   : va_getDriverName() returns 0
04-23 22:04:40.498  1453  3028 I libva   : Trying to open /system/lib/dri/i965_drv_video.so
04-23 22:04:40.498  1453  3028 I libva   : Found init function __vaDriverInit_0_33
04-23 22:04:40.499  1453  3028 I libva   : va_openDriver() returns 0

How could I enable more verbose log?